Microsoft.Recognizers.Text.NumberWithUnit.Tests/TestNumberWithUnitChinese.cs

```diff
@@ -16,10 +15,20 @@ private void BasicTest(IModel model, string source, string value)
             Assert.AreEqual(value, resultJson.First().Resolution["value"] + " " + resultJson.First().Resolution["unit"]);
         }
 
-        private void MultiTest(IModel model, string source, int count)
+        private void BasicTest(IModel model, string source, string[] values)
         {
-            var result = model.Parse(source);
-            Assert.AreEqual(count, result.Count);
+            var results = model.Parse(source);
+            Assert.AreEqual(values.Length, results.Count);
+            var resultsValues = results.Select(x => GetStringValue(x)).ToArray();
+            CollectionAssert.AreEqual(values, resultsValues);
+        }
+
+        private string GetStringValue(ModelResult source)
+        {
+            object value, unit;
+            source.Resolution.TryGetValue(nameof(value), out value);
```

Comment:
Do you want to do TryGetValue here?  Wouldn't you want it to tell you if not present or do you want the null further downstream?